transport/transporters/kafka/config.go

```diff
@@ -36,6 +37,12 @@ var Flags = []cli.Flag{
 		Usage:  "Kafka topic name ",
 		EnvVar: "BIFROST_KAFKA_TOPIC",
 	},
+	altsrc.NewIntFlag(cli.IntFlag{
+		Name:   ConfVarKafkaPartitionCount,
+		Usage:  "Number of Kafka partitions",
+		EnvVar: "KAFKA_PARTITION_COUNT",
```

Comment:
Is it possible for the client to look it up?

Reply:
Same as above comment, I can remove this now.